index.js

```diff
@@ -80,6 +81,16 @@ module.exports = async (inputs, options = {}) => {
 				return;
 			}
 
+			if (input === 'node') {
+				const processes = await psList();
+				processes.forEach(async ps => {
```

Comment:
This should use `Promise.all` and be awaited so it doesn't cause unhandled rejections if `kill()` fails.